packages/repack/src/commands/common/runAdbReverse.ts

```diff
@@ -1,35 +1,54 @@
+import path from 'node:path';
 import execa from 'execa';
 import type { Logger } from '../../types';
 
 interface RunAdbReverseParams {
+  logger?: Logger;
   port: number;
   verbose?: boolean;
-  logger?: Logger;
+  wait?: boolean;
+}
+
+function getAdbPath() {
+  const androidHome = process.env.ANDROID_HOME;
+  return androidHome ? path.join(androidHome, 'platform-tools', 'adb') : 'adb';
+}
+
+async function waitForDevice() {
+  const adbPath = getAdbPath();
+  const command = `${adbPath} wait-for-device`;
```

Comment:
this waits for the device to be opened, but not booted yet. Test if that's enough for adb reverse to work

Reply:
it did work in testers without a hitch 👍

Reply:
my theoery is that `adb reverse` only interacts with the network layer of the Android VM and not the OS so no need for waitining for booted state

Reply:
also this will fail when more than 1 device is connected

Reply:
What's the failure scenario here? That it will fail to do adb reverse on the correct device?

Reply:
both `adb reverse` and `adb wait-for-device` can fail with more than 1 available, I've addressed this in the newest commit